### PR TITLE
OLMIS-598: Added unit test exemplar and code coverage report

### PR DESCRIPTION
**Pull request by lkukielka**
This change is [Reviewable](https://reviewable.io/reviews/openlmis/openlmis-requisition/3)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -48,6 +49,28 @@ flyway {
     placeholderSuffix = ']'
 }
 
+sourceSets {
```
Comment (joshzamor) [inline build.gradle]: Is this necessary?  I thought Gradle would identify these source sets by default.

**Comment by joshzamor** [comment]
Please add to the README on how to:  run tests, location of test report(s).

**Comment by joshzamor** [comment]
LGTM once merge conflicts resolved.

<details><summary><i>Previously, joshzamor (Josh Zamor) wrote…</i></summary>
> Please add to the README on how to:  run tests, location of test report(s).
</details>


---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openlmis/openlmis-requisition/3#-:-KJqYCrQmx1gvGikDFKC:661425463)*

**Comment by pgesek** [comment]
Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openlmis/openlmis-requisition/3)*

**Pull request merged**